library/appointments.inc.php

```diff
@@ -41,6 +41,42 @@
     'trackerstatus' => array( 'trackerstatus', 'date', 'time', 'patient' ),    
 );
 
+/*Arrays for the interpretation of recurrence patterns.*/
+$REPEAT_FREQ = array(
+	'1' => xl('Every'),
+	'2' => xl('Every') . " " . xl('2nd'),
+	'3' => xl('Every') . " " . xl('3rd'),
+	'4' => xl('Every') . " " . xl('4th'),
+	'5' => xl('Every') . " " . xl('5th'),
+	'6' => xl('Every') . " " . xl('6th')
```

Comment:
much better off combining these constants or else will never be translated properly:
``````
'2' => xl('Every 2nd'),
...